example_data/import.sql

```diff
@@ -0,0 +1,2 @@
+\copy nw.view_node_wkt FROM '/data/node.csv' CSV HEADER;
```

Comment:
Consider adding `BEGIN;` and `COMMIT;` or maybe use `psql -1` in `docker-compose.test.yml`.

Reply:
I think I'll go with the latter. I might test running this as well as other SQL scripts ad-hoc the same way I'm already used to - start a transaction on the console, then run a script, see if stuff works or not, then `COMMIT` or `ROLLBACK` depending on the result (usually `ROLLBACK` if just testing things quickly and reverting to the original state). If possible, I'll aim at designing every SQL script such that they do not contain transaction commands but these commands are invoked either interactively or by a top-level script.